packages/utils/tokens/src/border-tokens.ts

```diff
@@ -1,11 +1,11 @@
 import { ViewStyle } from 'react-native';
 import { IOperationSet } from '@uifabricshared/foundation-tokens';
-import { ITheme } from '@uifabricshared/theming-ramp';
+import { ColorValue, ITheme } from '@uifabricshared/theming-ramp';
```

Comment:
nit: Import from `@fluentui-react-native/theme-types` or just `react-native`

Reply:
Latest commit addresses this.